crates/typst-library/src/foundations/array.rs

```diff
@@ -174,15 +174,25 @@ impl Array {
     /// Returns the first item in the array. May be used on the left-hand side
     /// of an assignment. Fails with an error if the array is empty.
     #[func]
-    pub fn first(&self) -> StrResult<Value> {
-        self.0.first().cloned().ok_or_else(array_is_empty)
+    pub fn first(
+        &self,
+        /// A default value to return if the index is out of bounds.
```

Comment:
```suggestion
        /// A default value to return if the array is empty.
```
There's no index here, so how about this?